packages/addons/addon-depends/system-tools-depends/lm_sensors/package.mk

```diff
@@ -4,7 +4,7 @@
 PKG_NAME="lm_sensors"
 PKG_VERSION="3.6.0"
 PKG_SHA256="0591f9fa0339f0d15e75326d0365871c2d4e2ed8aa1ff759b3a55d3734b7d197"
-PKG_ARCH="arm x86_64"
+PKG_ARCH="arm aarch64 x86_64"
```

Comment:
isn't that basically "all" ? so rm the whole line ?

Reply:
Done